src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -46,6 +46,7 @@ Terminal::Terminal() :
     _altGrAliasing{ true },
     _blockSelection{ false },
     _selectionMode{ SelectionInteractionMode::None },
+    _isTargetingUrl{ false },

```

Comment:
Terminals don't necessarily "target" things; what does this mean?

Reply:
This means that the selection is currently targeting a URL. (I'll rename it)

Side note, there's _a lot_ of members that are specific to selection. It would be nice to pull all of these out into a class or struct like this...
```
struct SelectionState
{
   // also add declarations for SelectionAnchors, ExpansionMode, InteractionMode, Endpoint

   std::optional<SelectionAnchors> _selection;
    bool _blockMode;
    std::wstring _wordDelimiters;
    ExpansionMode _multiClickMode;
    InteractionMode _interactionMode;
    bool _isTargetingUrl;
    Endpoint _selectionEndpoint;
    bool _anchorInactiveSelectionEndpoint;
};
```
I _kinda_ want to pull all of `TerminalSelection.cpp` into its own class too but idk what the _right_ way to clean all of this up would be. The reason I bring this up is because I hate having to add the word "selection" to every var related to it (i.e. `_isTargettingUrl` --> `_selectionIsTargetingUrl`). Thoughts? Maybe this should be a task I file under CodeHealth?